.github/workflows/image-factory.yaml

```diff
@@ -278,7 +285,7 @@ jobs:
           password: ${{ secrets.REGISTRY_PASSWORD }}
 
       - name: build netcat
-        run: podman build -t quay.io/securesign/netcat:v1.0.0 -f netcat/Dockerfile
+        run: podman build -t quay.io/securesign/netcat:${{ env.NETCAT_VER }} -f netcat/Dockerfile
 
       - name: push netcat
-        run: podman push quay.io/securesign/netcat:v1.0.0
+        run: podman push quay.io/securesign/netcat:${{ env.NETCAT_VER }}
```

Comment:
@tommyd450 make sure the var name matches the run name. I think you are missing an '_'